packages/venia-concept/src/RootComponents/Product/Product.js

```diff
@@ -1,12 +1,71 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import { Query } from 'react-apollo';
+import gql from 'graphql-tag';
 import { bool, shape, number, arrayOf, string } from 'prop-types';
 
 import { addItemToCart } from 'src/actions/cart';
 import ProductFullDetail from 'src/components/ProductFullDetail';
 import getUrlKey from 'src/util/getUrlKey';
-import getProductDetail from '../../queries/getProductDetail.graphql';
```

Comment:
These queries appear to be cached at some layer, and I couldn't figure out how to bust that cache. Inline queries don't appear to suffer from this problem.

Reply:
It looks like there's a big problem with `babel-plugin-import-graphql` and it's documented here: https://github.com/detrohutt/babel-plugin-import-graphql/blob/master/README.md#install You have to clear the babel cache every time you change a query.

That's unsustainable--we may have drop this dependency and rely on `graphql-tag/loader` in Webpack instead. That'll be annoying, because then we also have to add [this Jest mapper](https://github.com/remind101/jest-transform-graphql). But it's worth it to avoid the bug.

Reply:
Good to know. Let's tackle separately. 👍